src/pages/privacy-policy.md

```diff
@@ -0,0 +1,189 @@
+
+If you request a copy of your information, you may be required to pay a
+statutory fee.
+
+If you withdraw your consent to the use of your personal information for
+purposes set out in our Privacy Policy, we may not be able to provide you with
+access to (parts of) our website, applications, and services.
+
+## Changes to our privacy policy
+
+We may change this privacy policy from time to time. Still, we will not reduce
+your rights under this privacy policy. We will always update the privacy policy
+on our website, so please try to read it when you visit the website (the “last
+updated” reference tells you when we last updated this privacy policy).
+
+## Cookies, analytics and traffic data
+
+Cookies are small text files that are transferred from our website and stored on
+your device. We use cookies to help us provide you with a personalized service,
+and to help make our website and service better for you.
+
+Our cookies may be session cookies (temporary cookies that identify and track
+users within our websites) which are deleted when you close your browser, or
+persistent cookies (cookies that enable our website to “remember” who you are
+and to remember your preferences within our website) which will stay on your
+computer or device after you close your browser).
+
+We use the following different types of cookies:
+
+### Performance cookies and analytics technologies
+
+Performance cookies collect information about how visitors and users use our
+website. These cookies register, for example, which functionality our visitors
+use most often, or if they receive error messages from certain areas of our
+website. These cookies don’t collect information that can be used to identify a
+visitor or user. All information collected by these cookies is aggregated and
+thus anonymous. We only use these cookies to improve how our website works.
+
+### Functionality cookies
+
+Functionality cookies allow our website to remember the choices you make (such
+as your user name, language or the region you are in) and provide enhanced and
+more personal features. These cookies can also be used to remember changes you
+have made to text size, fonts and other parts of web pages that you can
+customize. They may also be used to provide services you have asked for. The
+information collected by these cookies may be anonymized. They cannot track your
+browsing activity on other websites.
+
+### IP address and traffic data
+
+We keep a record of the traffic data that is logged automatically by our
+servers, such as your Internet Protocol (IP) address, device information, the
+website that you visited before ours and the website you visit after leaving our
+site. We also collect some site, application and service statistics, such as
+access rates, page hits and page views. We are not able to identify any
+individual from traffic data or site statistics.
+
+#### Google Analytics
+
+Our platform uses Google Analytics, which uses cookies. The information created
```

Comment:
AFAIK that's simply not true, or am I missing something?

Reply:
True! I copy-pasted, this should be changed! Opening another PR with a fix. Thanks for the find!